app/views/admin/editions/_image_template.html.erb

```diff
@@ -0,0 +1,82 @@
+<% edition.images.each_with_index do |image, index| %>
+  <div class="govuk-grid-row govuk-!-margin-bottom-6">
+    <h3 class="govuk-heading-m govuk-!-margin-bottom-4 govuk-!-padding-left-3"> <%= image.persisted? ? "Image #{index + 1}" : "New image"  %></h3>
+    <div class="govuk-grid-column-one-half">
+      <% if image.new_record? %>
+        <%= render "govuk_publishing_components/components/file_upload", {
+          label: {
+            text: "File",
+            bold: true,
+          },
+          name: "edition[images_attributes][#{index}][image_data_attributes][file]",
+          id: "edition_images_#{index}_image_data_file",
+        } %>
+      <% end %>
+
+      <% if image.image_data.file_cache.present? %>
+        <%= hidden_field_tag "edition[images_attributes][#{index}][image_data_attributes][file_cache]", image.image_data.file_cache %>
+        <p class="govuk-body"><%= "#{File.basename(image.image_data.file_cache)} already uploaded" %></p>
+      <% end %>
+
+      <%= render "govuk_publishing_components/components/input", {
+        label: {
+          text: "Alt text",
+          bold: true
+        },
+        name: "edition[images_attributes][#{index}][alt_text]",
+        id: "edition_images_#{index}_alt_text",
+        value: image.alt_text,
+        error_items: errors_for(image.errors, :alt_text),
+      } %>
+
+      <%= render "govuk_publishing_components/components/textarea", {
+        label: {
+          text: "Caption",
+          bold: true
+        },
+        name: "edition[images_attributes][#{index}][caption]",
+        id: "edition_images_#{index}_caption",
+        rows: 5,
+        value: image.caption,
+        error_items: errors_for(image.errors, :caption),
+      } %>
+
+      <% if image.persisted? %>
+        <%= hidden_field_tag "edition[images_attributes][#{index}][id]", image.id %>
+        <%= hidden_field_tag "edition[images_attributes][#{index}][_destroy]", "1" %>
+
+        <%= render "govuk_publishing_components/components/checkboxes", {
+          name: "edition[images_attributes][#{index}][_destroy]",
+          id: "edition_images_#{index}__destroy",
+          items: [
+            {
+              label: "(uncheck to remove)",
+              value: "0",
+              checked: true,
```

Comment:
With the hardcoded `checked: true` option, I think this means the checkboxes will become re-checked if there are other validation errors on the page.

I think the user's expectation would be that the state of these checkboxes remains intact.

Reply:
Good catch. Have fixed 👍 thanks for the review